packages/pipeline-editor/src/RuntimesWidget.tsx

```diff
@@ -28,12 +28,31 @@ import React from 'react';
 
 import { PipelineService, RUNTIMES_NAMESPACE } from './PipelineService';
 
+const RUNTIMES_METADATA_CLASS = 'elyra-metadata-runtimes';
+
+const addTrailingSlash = (url: string): string => {
+  return url.endsWith('/') ? url : url + '/';
+};
+
+const getGithubURLFromAPI = (apiEndpoint: string): string => {
```

Comment:
The  logic to derive the GUI URL needs to mirror https://github.com/elyra-ai/elyra/blob/master/elyra/util/git.py#L89 to handle GitHub Enterprise URL patterns that we are aware of.

Reply:
I was following the docs (https://docs.github.com/en/rest/reference/enterprise-admin), but I missed that there is another format than just the GitHub.com and Enterprise Server formats I covered. It seems IBM uses GitHib AE, I'll update this and remove the if statement and always remove `api.`